docs/expandable/vue.md

```diff
@@ -0,0 +1,65 @@
+## Import
+
+### Use in entire app
+
+```js
+import { Expandable } from '@warp-ds/vue';
+app.use(Expandable);
+```
+
+### Use in one component and special imports
+
+```js
+import { wExpandable, wExpandTransition, wWillExpand } from '@warp-ds/vue';
+```
+
+## Usage
+
+```js
+<w-expandable title="I am expandable">
+  <p>Hello there I am some informative content</p>
+</w-expandable>
+```
+
+## Visual options
+
+### Default
+
+```js
+<w-expandable title="I am expandable">
+  <p>Hello there I am some informative content</p>
+</w-expandable>
+```
+
+### Expandable box
+
+```js
+<w-expandable title="I am expandable" box>
+  <p>Hello there I am some informative content</p>
+</w-expandable>
+```
```

Comment:
It just occurred to me that these docs will not show the different visual options of components based on the code blocks, like they would in Fabric docs. Those visual representations of components are covered in the Example section of each page. My conclusion being, we can stick to **Usage** section and remove the **Visual options** section completely. Wdyt?

Reply:
That was my thought, its not ideal but It will have to do for now I think.    I think we need to take a look at the doc-component that shows live component (The block that has a theme dropdown now) Maybe add some support for types and variants in that(?) New task perhaps?

Reply:
yeah I agree, I also thought about that. But I think we can go through everything when all components are moved. We also need to ensure that the docs structure is the same, but it will be easier when we have something to look at and compare. So let's have a cleanup/improvement at a later stage soon?